src/lib.rs

```diff
@@ -527,7 +527,7 @@ impl<T: Clone + Num> Sub<Complex<T>> for Complex<T> {
 
 forward_all_binop!(impl Mul, mul);
 
-// (a + i b) * (c + i d) == (a*c - b*d) + i (a*d + b*c)
+// (a + i b) * (c + i d) + (a + i b) + (c + i d) == (a*c - b*d) + i (a*d + b*c) + i (b + d)
```

Comment:
It looks like this comment was meant for `MulAdd`, but I'm also confused why there are *two* addends, and reusing the same variables as the multiplicands.

Reply:
Thanks, this was not meant to be there at all.